todo_app/app.py

```diff
@@ -1,55 +1,118 @@
-from operator import itemgetter
+from operator import attrgetter
 from flask import Flask, render_template, request, redirect
-from todo_app.data import session_items
 from todo_app.flask_config import Config
+import os,requests
+from dotenv import load_dotenv
+from todo_app.ToDoItem  import  ToDoItem
 
+# Set app variables
+load_dotenv()
+key=os.getenv('TRELLO_KEY')
+token=os.getenv('TRELLO_TOKEN')
+trelloBoardID='60fad7c6c463fc540fbffb94'
+trelloBaseList='60fad7c6c463fc540fbffb95' # will place new cards in To Do list
 app = Flask(__name__)
 app.config.from_object(Config)
 
+# Dictionary used to map request calls 
+jsonDict={
+    "PUT":requests.put,
+    "POST":requests.post,
+    "GET":requests.get,
+    "DELETE":requests.delete
+ }
 
+# Common function used to request data from URL's
+def requestJson (method, url, args):
+    r = jsonDict[method](url, params=args) 
```

Comment:
Looks like you maybe just wanted to use `requests.request(method, url, params=args)`